README.md

```diff
@@ -9,3 +9,24 @@ might help with the notation for mathematical expressions.
 $G_1=(V_1 , E_1)$ is isomorphic to $G_2 = (V_2, E_2)$ if there exists a
 one-to-one and onto function (bijection) $f: V_1 \rightarrow V_2$ such that $(u,v)
 \in E_1$ iff $(f(u),f(v)) \in E_2$.
+
+A complete graph with n vertices needs to have every pair of vertices be connected by an edge.
+$E = (n/2) = (n(n-1))/2$
+
+$G_1$ and $G_2$ have the same amount of vertices since each vertex in $G_1$ paired with a vertex in $G_2$ since it's stated in the bijection ($f: V_1 \rightarrow V_2$).
```

Comment:
How do you know that this bijection exists? One way of showing that is to give a way to construct it.